src/toTransform.js

```diff
@@ -64,20 +104,13 @@
   }
 
   function toTransform (properties) {
-    var results = [];
-    var conversion = convertTranslate(properties.translate);
-
-    if (conversion !== null) {
-      results.push(conversion);
-    }
-
-    conversion = convertScale(properties.scale);
-
-    if (conversion !== null) {
-      results.push(conversion);
-    }
-
-    return results.join(' ');
+    return [
+      convertTranslate(properties.translate),
+      convertRotate(properties.rotate),
+      convertScale(properties.scale)
```

Comment:
All three together yay! Let's add tests for combinations of them in a separate patch.